javascript-functions/main.js

```diff
@@ -0,0 +1,41 @@
+function convertMinutesToSeconds(minutes) {
+  var convert = minutes * 60;
+  return convert;
+}
+
+var convertMinutesToSecondsResult = convertMinutesToSeconds(5);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function greet(name) {
+  var greeting = '"' + 'Hey, ' + name + '"';
+  return greeting;
+}
+
+var greetResult = greet('Nathan');
+console.log('greetResult:', greetResult);
+
+function getArea(width, height) {
+  var area = width * height;
+  return area;
+}
+
+var getAreaResult = getArea(17, 42);
+console.log('getAreaResult:', getAreaResult);
+
+function getFirstName({ firstName, lastName }) {
```

Comment:
The approach you are taking is called destructuring. I'm not sure if that is intentional as it is typically a bit more advanced. If you stuck with this approach then you could simplify the code block to:

```
function getFirstName({firstName, lastName}) {
  return firstName
}
```

Or, if you weren't to use destructuring it would look like:

```
function getFirstName(person) {
  var firstName = person.firstName;
  return firstName;
}
```

Reply:
Either way, there is no need to create a new JavaScript object in the code block :)